waspc/examples/todoApp/src/client/pages/ProfilePage.tsx

```diff
@@ -13,10 +15,36 @@ export const ProfilePage = ({
 }: {
   user: User
 }) => {
+  const [messages, setMessages] = useState<{ id: string, username: string, text: string }[]>([]);
+  const { socket, isConnected } = useSocket<ServerToClientEvents, ClientToServerEvents>()
+  const inputRef = useRef<HTMLInputElement>(null)
```

Comment:
I know this is just an example app but I've seen the `useState` pattern with inputs more often:
```
const [message, setMessage] = useState('');
...
<input value={value} onChange={event => setMessage(event.target.value)} />
```

Reply:
On change will fire for every keystroke though, right? Also, I was wanting to handle both enter to submit and a button style.